### PR TITLE
fix samples

### PR DESCRIPTION
**Pull request by billba**

### CODE REVIEW

File: samples/backchannel/index.html [hunk 1]
```diff
@@ -65,7 +65,7 @@ <h2 style="font-family:Segoe UI;">Type a color into the WebChat!</h2>
                 secret: params['s'],
                 token: params['t'],
                 domain: params['domain'],
-                webSocket: params['webSocket'] // defaults to true
+                webSocket: params['webSocket'] && params['webSocket'] === "true" // defaults to true
```
Comment (danmarshall) [inline samples/backchannel/index.html]: Or you can do the inline boolean conversion !!params['webSocket']

Comment (billba) [inline samples/backchannel/index.html]: No because I want to pass in `undefined` in the (common) case where the `webSocket` parameter doesn't exist.

Comment (danmarshall) [inline samples/backchannel/index.html]: Does the && convert to boolean?

Comment (billba) [inline samples/backchannel/index.html]: Anyway, `!!"false" === true` which is not what we want.

Comment (billba) [inline samples/backchannel/index.html]: No.

Comment (danmarshall) [inline samples/backchannel/index.html]: Hold up. So what if someone passes in "false" on the url? Do you get the desired behavior?

Comment (billba) [inline samples/backchannel/index.html]: Yes.

**Pull request merged**